codespell_lib/data/dictionary.txt

```diff
@@ -32404,12 +32442,20 @@ manetainer->maintainer
 manetainers->maintainers
 manetaining->maintaining
 manetains->maintains
+mangae->manage
```

Comment:
Also `manga` and `mange`, and `mangle` although the keys **A** and **L** are far apart?

Reply:
I'll add `manga`. I couldn't find `mange` and `mangle`.